spyder/workers/updates.py

```diff
@@ -59,7 +60,7 @@ def check_update_available(self, version, releases):
 
     def start(self):
         """Main method of the WorkerUpdates worker"""
-        self.url = 'https://api.github.com/repos/spyder-ide/spyder/releases'
+        self.url = 'https://repo.continuum.io/pkgs/main/win-64/repodata.json'
```

Comment:
This looks good, thanks @dalthviz! Two comments:

1. We need to use this only for Anaconda, and use the old method for pip.
2. The repodata.json needs to be pulled from the channel corresponding the operating system in which Spyder is running.